server/client/src/Pages/Homepage.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+const Home = () => {
+    const navigate = useNavigate();
+
+    const goToLogin = () => {
+        navigate('/login');
+    };
+
+    const goToSignup = () => {
+        navigate('/register');
+    };
+
+    return (
+        <div className="flex flex-col justify-center items-center h-screen bg-gray-100">
+            <h1 className="text-4xl font-bold mb-8 text-gray-800">Welcome to the Virtual Event Platform</h1>
+            <div className="space-y-4">
+                <button
```

Comment:
Button ki jagah Link tag use karna makes more sense kyunki hai to link hi dusre page pe le jaega

Reply:
Abhi home page ke liye aese hi de diye hai baad me home page jab banega to hatt hi jayega